tests/test_edgeql_expressions.py

```diff
@@ -3058,7 +3058,7 @@ async def test_edgeql_expr_introspect_bad_01(self):
     async def test_edgeql_expr_introspect_bad_02(self):
         with self.assertRaisesRegex(
                 edgedb.QueryError,
-                r'cannot introspect transient type variant'):
+                r"type 'A' does not exist"):
```

Comment:
I think this an improvement to the error message, because a "transient type variant" is not something that we document anywhere.

Saying that "type 'A' does not exist" hints that "A :=" does not declare a type, but just a value binding. So you have to do `INTROSPECT TYPEOF A` instead.